.travis.yml

```diff
@@ -37,12 +40,17 @@ jobs:
         - date
         - psql -U postgres -c "create user test with password 'test';"
         - psql -c 'create database test owner test;' -U postgres
+        - docker run -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=SqlPassword234" -p 1434:1433 --name=mssql-server-linux-latest -e TZ=America/Los_Angeles -d mcr.microsoft.com/mssql/server:2019-latest
+        - docker cp sqlserver.sql mssql-server-linux-latest:sqlserver.sql
+        - sleep 15
```

Comment:
This doesn't look robust. Some earlier work used some scripts to detect once the database is up and running (basically polling every second or something like that).